chpl/chpl-service/src/test/java/gov/healthit/chpl/dao/impl/StatisticsDAOTest.java

```diff
@@ -83,23 +85,23 @@ public static void setUpClass() throws Exception {
     }
 
     /**
-     * Given that getTotalDevelopers(DateRange) is called 
-     * When the start date is set to the beginning of time and endDate is set to now()
-     * Then all non-deleted developers are returned
+     * Given that getTotalDevelopers(DateRange) is called When the start date is
```

Comment:
The Given-When-Then syntax is a feature of our acceptance criteria. I'd rather have it look right here and be flagged for checksyle than vice versa